### PR TITLE
New placeholder field for interactives

### PR DESCRIPTION
**Pull request by Mullefa**
See in-line comment for why this field is being added.

### CODE REVIEW

File: thrift/src/main/thrift/atoms/interactive.thrift [hunk 1]
```diff
@@ -11,4 +11,11 @@ struct InteractiveAtom {
   4: required string html
   5: optional string mainJS
   6: optional string docData
+
+  /*
+  When an interactive is within an amp-iframe,
+  a placeholder field is required to for the iframe to be displayed at the top of the AMP.
+  See here: https://amp.dev/documentation/components/amp-iframe/ for more information.
+   */
+  7: optional string placeholder
```
Comment (Mullefa) [inline thrift/src/main/thrift/atoms/interactive.thrift:20]: Any opinions over `placeholder` vs `placeholderUrl` as the field name?

Comment (juanvaccari) [reply thrift/src/main/thrift/atoms/interactive.thrift:20]: if it is an url, probably better placeholderUrl I think... but also you could say it is for the amp-iframe in the name maybe? Or is it obvious already? ampIframeUrl? or ampIframePlaceholderUrl?

Comment (Mullefa) [reply thrift/src/main/thrift/atoms/interactive.thrift:20]: Hmm - I was talking to @ajwl about including amp in the name. Probably splitting hairs, but I'm not sure we need to be that specific. If I understand correctly, it's a url for a placeholder image that will be used by amp, but there's nothing amp specific about it (?)

Comment (juanvaccari) [reply thrift/src/main/thrift/atoms/interactive.thrift:20]: sure, if it is not specific to this, fair enough!

Comment (JustinPinner) [reply thrift/src/main/thrift/atoms/interactive.thrift:20]: Agree that `placeholderUrl` is probably the best fit for this

**Comment by Mullefa** [comment]
Ok, field name changed to `placeholderUrl`. Ready to review / merge / release.

**Review by juanvaccari — APPROVED** [review]

**Review by JustinPinner — APPROVED** [review]
LGTM 👍

**Pull request merged**